### ISSUE TITLE
Clarification on Validation and Test Set Usage in UCR/UEA Datasets

### ISSUE THREAD

**Issue by ZLiu21** [chenguolin/NuTime#2]
Dear authors,  

I am analyzing the data loading process in NuTime and noticed that in `NuTime/src/data/dataloader.py`, the dataloader appears to use the test set as the validation set for datasets in the UCR and UEA archives. Specifically:  

- Line 45:  
  ```python
  transform_dict['val'] = test_transform
  ```  
- Lines 75-81:  
  ```python
  # no validation set for some datasets
  try:
      config.no_validation_set = False
      dataset_dict['val'] = CustomDataset(config=config, type='val', transform=transform_dict['val'])
  except:
      config.no_validation_set = True
      dataset_dict['val'] = CustomDataset(config=config, type='test', transform=transform_dict['test'])
  ```  
This suggests that if a dataset lacks a predefined validation set, the test set is used instead. I would like to confirm:  

When evaluating NuTime on UCR and UEA datasets, is the final test performance in the main text of NuTime based on the best model selected according to this "validation" set (which is actually the original test set)?  

Looking forward to your clarification. Thanks for your time!
